package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "event-manager",
+  "version": "1.0.0",
+  "description": "This App allows client post events online and also grants admin access level to the site administrators to smoothly run the the site.",
+  "main": "server/app.js",
+  "scripts": {
+    "start": "babel-watch server/app.js",
+    "babel-node": "babel-node --",
+    "build": "./node_modules/.bin/babel server -d build",
+    "dist": "./node_modules/.bin/babel server -d dist --presets=babili"
```

Comment:
What is this also used for??

Reply:
Its not necessary, initially I was trying to play around running my code from a build folder, I'll adjust that